### ISSUE TITLE
`allowUnusedLabels` should be false in the --init configuration

### ISSUE THREAD

**Issue by RyanCavanaugh** [microsoft/TypeScript#38304]
**TypeScript Version:**  nightly

**Search Terms:** allowUnusedLabels

**Code**

Compile with `tsc --init`
```ts
console.log("a", "b"), { value: 3 };
```

**Expected behavior:** This should be an error;  the closing paren was misplaced

**Actual behavior:** No error

**Playground Link:** https://www.typescriptlang.org/play?#code/MYewdgziA2CmB00QHMAUAiAhugNAAnQCN0BKfAbzwDdNoBXWALjwGY8BfAbgCgg

**Related Issues:** No
